tui_main.c

```diff
@@ -1903,6 +1908,17 @@ int main(int argc, char** argv) {
         exit(1);
       }
     } break;
+    case Argopt_bpm: {
+      init_bpm = atoi(optarg);
+      if (init_bpm < 0 ||
```

Comment:
Allowing BPM to be set to 0 will cause the event loop to lock up, freezing the application and terminal until the user is able to switch to a different virtual terminal and kill the application.